pkg/blkid/blkid.go

```diff
@@ -54,3 +55,19 @@ func (di *DeviceIdentifier) GetOnDiskFileSystem() string {
 
 	return C.GoString(fstype)
 }
+
+func (di *DeviceIdentifier) GetPartitionUUID() string {
```

Comment:
Can we create a common function that can be used by both GetPartitionUUID and GetOnDiskFIlesystem.

eg :

```golang
func X(typeIdentifier string) {
}

func (di *DeviceIdentifier) GetOnDiskFileSystem() string {
    return X(fsTypeIdentifier)
}

func (di *DeviceIdentifier) GetPartitionUUID() string {
    return X(partitiondUUIDIdentifier)
}
```